db/models/NewsArticle.js

```diff
@@ -15,7 +15,7 @@ const NewsArticleSchema = new Schema({
     type: String,
     required: true,
   },
-  assosiated_press: {
+  assosiatedPress: {
```

Comment:
you dont need to change this,becouse it is not going to work, in mongodb  Atlas another field name